### DISCUSSION TITLE
Where to find the 'automatic page generator' part of the repository?

### DISCUSSION THREAD

**Discussion by SrivalliAlla** [community/community#63333] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi! I am trying to do a course on Courera.com and need to find the 'launch automatic page generator' button on the repository I created for the course. I am unable to find it in the settings.

**Comment by C0RE1312** [comment, marked as answer]
"There is no automatic page generator button in GitHub anymore, I am not able to find it in any of my repos. GitHub now uses Jekyll to make the pages."
Checkout this: https://stackoverflow.com/questions/41248716/cant-locate-automatic-page-generator-button-in-github
